backend/src/test/java/com/cruru/applicant/service/ApplicantServiceTest.java

```diff
@@ -47,104 +42,82 @@ class ApplicantServiceTest extends ServiceTest {
-    //
-    //        // then
-    //        assertAll(
-    //                () -> assertThat(process.getId()).isEqualTo(processResponse.id()),
-    //                () -> assertThat(process.getName()).isEqualTo(processResponse.name()),
-    //                () -> assertThat(applicant.getId()).isEqualTo(applicantResponse.id()),
-    //                () -> assertThat(applicant.getName()).isEqualTo(applicantResponse.name()),
-    //                () -> assertThat(applicant.getEmail()).isEqualTo(applicantResponse.email()),
-    //                () -> assertThat(applicant.getPhone()).isEqualTo(applicantResponse.phone())
-    //        );
-    //    }
-
     @DisplayName("id에 해당하는 지원자가 존재하지 않으면 Not Found 예외가 발생한다.")
     @Test
     void findById_notFound() {
         // given
         long invalidId = -1L;
 
         // when&then
-        assertThatThrownBy(() -> applicantService.findById(invalidId))
-                .isInstanceOf(ApplicantNotFoundException.class);
+        assertThatThrownBy(() -> applicantService.findById(invalidId)).isInstanceOf(ApplicantNotFoundException.class);
     }
 
-    @DisplayName("id로 지원자의 상세 정보를 찾는다.")
+    @DisplayName("지원자의 정보 변경 요청의 이름, 이메일, 전화번호 변경점이 없다면 예외를 던진다")
     @Test
-    void findDetailById() {
+    void updateApplicantInformation() {
         // given
-        Dashboard dashboard = createBackendDashboard();
-        dashboardRepository.save(dashboard);
-        Process process = createFirstProcess(dashboard);
-        processRepository.save(process);
-        Applicant applicant = createPendingApplicantDobby(process);
-        applicantRepository.save(applicant);
-
-        Question question = questionRepository.save(createShortAnswerQuestion(null));
-        questionRepository.save(question);
-        Answer answer = new Answer("토끼", question, applicant);
-        answerRepository.save(answer);
+        Applicant applicant = ApplicantFixture.createPendingApplicantDobby();
+        String originalName = applicant.getName();
+        String originalEmail = applicant.getEmail();
+        String originalPhone = applicant.getPhone();
+        ApplicantUpdateRequest noChangeRequest = new ApplicantUpdateRequest(originalName, originalEmail, originalPhone);
 
         // when
-        ApplicantDetailResponse applicantDetailResponse = applicantService.findDetailById(applicant.getId());
-
-        //then
-        List<QnaResponse> qnaResponses = applicantDetailResponse.qnaResponses();
-        assertAll(
-                () -> assertThat(qnaResponses.get(0).question()).isEqualTo(question.getContent()),
-                () -> assertThat(qnaResponses.get(0).answer()).isEqualTo(answer.getContent())
-        );
+        Applicant savedApplicant = applicantRepository.save(applicant);
+
+        // then
+        assertThatThrownBy(() -> applicantService.updateApplicantInformation(noChangeRequest,
+                savedApplicant
+        )).isInstanceOf(ApplicantNoChangeException.class);
```

Comment:
```suggestion
        assertThatThrownBy(() -> applicantService.updateApplicantInformation(noChangeRequest, savedApplicant))
                .isInstanceOf(ApplicantNoChangeException.class);
```
개행이 이렇게 들어가야 하지 않을까요?